docs/streams/publish-messages.md

```diff
@@ -0,0 +1,215 @@
+---
+sidebar_position: 60
+title: Publish Messages to Streams
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to publish messages to a stream in Macrometa. Before publishing messages, you must first [create a stream](create-streams.md) to publish the messages to.
```

Comment:
```suggestion
This page explains how to publish messages to a stream in Macrometa. Before publishing messages, you must first [create a stream](create-streams.md) to which you want to publish messages.
```

Probably not a necessary change, but I prefer to avoid ending sentences with "to"

Reply:
The stream will be created automatically if does not exist yet. So, the step `create-streams.md` is redundant.